deploy/src/deploymentUtils.js

```diff
@@ -10,23 +10,27 @@ const {
   FOREIGN_RPC_URL,
   HOME_RPC_URL,
   GAS_LIMIT,
-  GAS_PRICE,
+  HOME_GAS_PRICE,
+  FOREIGN_GAS_PRICE,
   GET_RECEIPT_INTERVAL_IN_MILLISECONDS
 } = require('./web3')
 
 async function deployContract(contractJson, args, { from, network, nonce }) {
   let web3
   let url
+  let gasPrice
   if (network === 'foreign') {
     web3 = web3Foreign
     url = FOREIGN_RPC_URL
+    gasPrice = FOREIGN_GAS_PRICE
   } else {
     web3 = web3Home
     url = HOME_RPC_URL
+    gasPrice = HOME_GAS_PRICE
   }
   const options = {
     from,
-    gasPrice: GAS_PRICE
+    gasPrice
```

Comment:
It should be added to the call `sendRawTx` below. The dictionary `options` is not passed to `sendRawTx` at all.

Reply:
Done, thanks. The previous version wasn't using the gasPrice either, so this might've been a bug that we didn't notice before.